### ISSUE TITLE
Poly's trying to reclaim things that were reclaimed

### ISSUE THREAD

**Issue by CitrusWire** [Anuken/Mindustry#3757]
120.3 - Windows - Overgrowth map

Poly's are trying to rebuild/rebuild things that were reclaimed before the Poly's were even created. Also things that were never "mine" to begin with.

![image](https://user-images.githubusercontent.com/72134064/101091833-a2655c00-35b0-11eb-9882-34f778979eb1.png)
[sector-serpulo-134.zip](https://github.com/Anuken/Mindustry/files/5639091/sector-serpulo-134.zip)

**Comment by Anuken** [comment]
> Poly's are trying to rebuild/rebuild things that were reclaimed before the Poly's were even created.

When they were created is irrelevant. It would be awful if they only rebuilt things broken after they were made.

> Also things that were never "mine" to begin with.

If you begin deconstructing a block, it automatically switches to your team, regardless of whether it was derelict before. If that is broken, they will attempt to rebuild it like they would any other block.

If you don't want this, simply remove the ghost blocks like you would any other block.

**Comment by CitrusWire** [comment]
> 
> 
> > Poly's are trying to rebuild/rebuild things that were reclaimed before the Poly's were even created.
> 
> When they were created is irrelevant. It would be awful if they only rebuilt things broken after they were made.

Ok. I included that just to try and help narrow down what may be the cause.

> If you don't want this, simply remove the ghost blocks like you would any other block.

There were no ghost blocks. At least, none I could see. I only saw what's in the screenshot. Typically if I reclaim a neutral thing a Poly will help deconstruct it, then the neutral thing completely disappears - no ghost blocks. In this case the things were removed before the Poly's existed and then for whatever reason the Poly's did whatever that thing is - they're neither building nor removing things, just pointing at that space.
